pkg/apis/experiment/query.go

```diff
@@ -0,0 +1,73 @@
+package experiment
+
+const (
+	SaveExperimentQuery = `mutation saveChaosExperiment($projectID: ID!, $request: SaveChaosExperimentRequest!) {
+                      saveChaosExperiment(projectID: $projectID, request: $request)
+                     }`
+
+	GetExperimentQuery = `query listExperiment($projectID: ID!, $request: ListExperimentRequest!) {
```

Comment:
Can you check if all the fields that are being fetched in API are required? whichever are not required we can remove, it will reduce network load